tools/packaging/kernel/patches/5.10.x/0001-arm64-sparsemem-reduce-SECTION_SIZE_BITS.patch

```diff
@@ -0,0 +1,94 @@
+From 5745513881520f48401ac40306ac8cd6ef35beaf Mon Sep 17 00:00:00 2001
```

Comment:
@jongwu shouldn't the patch be for `tools/packaging/kernel/patches/6.1.x` rather than `tools/packaging/kernel/patches/5.10.x`?

Reply:
It's a fixup in kernel part and I think it is more elegant than a fix in qemu side.

Reply:
oops, yes. Sorry.